bears/general/IndentationBear.py

```diff
@@ -291,14 +291,14 @@ def get_valid_sequences(file, sequence, annotation_dict):
 
             # ignore if within string
             for string in annotation_dict['strings']:
-                if(sequence_position >= string.start and
-                   sequence_position <= string.end):
+                if(gt_eq(sequence_position, string.start) and
```

Comment:
yeah the order of functions inside a module or class in arbitrary. I think that's because of modules being imported first and executed afterwards.